mbed_greentea/mbed_test_api.py

```diff
@@ -269,6 +272,9 @@ def get_binary_host_tests_dir(binary_path, level=2):
         if run_app:
             cmd += ["--run"]    # -f stores binary name!
 
+    if tags:
+        cmd += ["--tag-filters", str(tags)]
```

Comment:
looking at the code in more depth, it looks like you don't even need the `str` wrapper, as you are not parsing this argument in this command line app.

e.g.
```python
if tags:
    cmd += ["--tag-filters", tags]
```
should work correctly.